### PR TITLE
Internet connectivity: set up file structure

### PR DESCRIPTION
**Pull request by user-don**
Set up file structure for API work and also put MainActivity.java under /ui/activities
(https://www.bloco.io/blog/2015/how-to-organize-your-android-code-structu
re)

### CODE REVIEW

File: app/src/androidTest/java/advancetransit/advancetransit/ApplicationTest.java [hunk 1]
```diff
@@ -8,6 +8,8 @@
  */
 public class ApplicationTest extends ApplicationTestCase<Application> {
     public ApplicationTest() {
+        // calls constructor of the parent class (ApplicationTestCase) with the Application
```
Comment (user-don) [inline app/src/androidTest/java/advancetransit/advancetransit/ApplicationTest.java]: added comments here to give more context based on online search of what this testing method does

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -8,7 +8,7 @@
         android:label="@string/app_name"
         android:theme="@style/AppTheme" >
         <activity
-            android:name=".MainActivity"
+            android:name=".ui.activities.MainActivity"
```
Comment (user-don) [inline app/src/main/AndroidManifest.xml]: moved MainActivity into /ui/activities

File: app/src/main/java/advancetransit/advancetransit/api/gtfs/CallGtfsApi.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package advancetransit.advancetransit.api.gtfs;
```
Comment (user-don) [inline app/src/main/java/advancetransit/advancetransit/api/gtfs/CallGtfsApi.java]: created two new java classes for calling APIs

**Comment by rcgheorghe** [comment]
this seems fine. I'm good with merging with master

**Comment by nickg413** [comment]
Let's merge it!

**Comment by user-don** [comment]
awesome

**Pull request merged**